packages/dropdowns/src/elements/Menu/Menu.tsx

```diff
@@ -22,18 +22,23 @@ import {
 import { MenuContext } from '../../utils/useMenuContext';
 
 interface IMenuProps extends HTMLAttributes<HTMLUListElement> {
-  /** See Popper [documentation](https://popper.js.org/docs/v2/modifiers/) for details */
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) modifiers */
   popperModifiers?: Modifiers;
+  /** Determines if events are enabled */
   eventsEnabled?: boolean;
+  /** Sets the z-index */
   zIndex?: number;
-  /**
-   * These placements differ from the default naming of Popper placements to
-   * accommodate RTL layouts
-   **/
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) placement. This property differs
+   * from the default Popper placements and accommodates RTL layouts.
+   */
```

Comment:
I don't think anything about Popper needs to be stated here. Garden has its own placement mechanism. The fact that Popper is used underneath is an implementation detail.

Compare with https://github.com/zendeskgarden/react-components/pull/931/files#r518914907

Reply:
Fixed in latest push.